ptx/S_12_6_Curl.ptx

```diff
@@ -5,65 +5,315 @@
       <ul>
         <li>
           <p>
-          How can you measure the rotational strength of a vector field?
+          What is meant by rotation of a vector field in a plane? 
           </p>
         </li>
         <li>
           <p>
-            Another???
+            How can we generalize our two dimensional measurement of rotation to work in three dimensions?
+          </p>
+        </li>
+        <li>
+          <p>
+            How can you measure the rotational strength of a vector field?
           </p>
         </li>
       </ul>
     </objectives>
     <introduction>
     <p>
-      Introduce the idea of a spinner (pinwheel) as a measure for how a vector field rotates (as a local measurement)
+      In <xref ref="S_12_5_Div">Section</xref> we examined two (and higher) dimensional vector field for how the strength of the vector field changed in different regions. In particular, we developed the <xref ref="Div_defn">divergence (Definition</xref>) of a vector field as a local (or density) measurement for how the strength of the vector field changes. In many physical settings it is also useful to measure the rotational strength of a vector field at a local scale. For instance, the vector field on the left of <xref ref="Curl_intro_vfs">Figure</xref> shows a vector field which flows in a counterclockwise fashion around the origin. The vector field on the right of <xref ref="Curl_intro_vfs">Figure</xref> shows a vector field which does not have a rotational aspect to its flow.
     </p>
-
+      <figure xml:id="Curl_intro_vfs">
+        <caption><p>Examples of vector fields with global rotation properties and without rotation in either the global or local sense.</p></caption>
+        <sidebyside widths="45% 45%">
+          <image source="images/fig_12_4_ex2_field1"/>
+          <image source="images/Div_activity_-x-y"/>
+        </sidebyside>
+      </figure>
+      <p>
+        These global ideas of rotation are nice but are not always visually apparent or may only appear in some regions. For instance, the vector field on the left of <xref ref="Curl_intro_vfs2">Figure</xref> seems to have rotation around at least a couple of different points in a global sense. The vector field on the right of <xref ref="Curl_intro_vfs2">Figure</xref> has the rotation of the flows dependant on which quadrant you are examining.
+      </p>
+      <figure xml:id="Curl_intro_vfs2">
+        <caption><p>Examples of vector fields with global rotation properties and without rotation in either the global or local sense.</p></caption>
+        <sidebyside widths="45% 45%">
+          <image source="images/Div_activity_xy1x"/>
+          <image source="images/fig_12_5_vf1"/>
+        </sidebyside>
+      </figure>
     </introduction>
     <activity xml:id="Curl_PA">
```

Comment:
Since this is the Preview, it should be inside the `introduction` and coded as `exploration` not `activity`. (That should be as simple as changing the open/closing tags and moving `</introduction>`.